### ISSUE TITLE
Dark theme icons clipping and no margin

### ISSUE THREAD

**Issue by JeFawk** [Scirra/Construct-3-bugs#523]
## Problem description

When collapsing the object part within a condition/action, it collapses too much and leaves the icon with no right margin, and even clips over some of them.

## Attach a .c3p

*A minimal Construct 3 project (.c3p) is required to be attached. Please see the guidelines*
![image](https://user-images.githubusercontent.com/7268697/27234761-d518ae84-52c6-11e7-9413-8ea26840cf4d.png)

## Expected result

I expect a minimum margin.

## Affected browsers

* Chrome: yes/
## System details

*If you see a crash report dialog, please copy and paste it to where it says "PASTE HERE" below. Otherwise please go to **Menu** -> **About** -> **Platform information** and paste that information there instead.*

<details><summary>View details</summary>

PASTE HERE

</details>

**Comment by Animmaniac** [comment]
![events-actionnameminwidth](https://user-images.githubusercontent.com/29487913/27237891-b0fa2e50-52c2-11e7-8a38-4dcabe8136c5.png)

This happens because I added a bit more padding to the left of conditions/actions names to separate them better (see figure 1). It seems the minimum size for actionNameCell or conditionNameCell is hard-coded somewhere.  If I set a min-width for actionNameCell it doesn't seem to be respected (the dividing border also disappears when collapsed).

To solve it I can reset the padding to the original value (figure 2) at the expense of a more balanced layout (although I think someone else will run into the same problem when customizing themes), or the resizing for actionNameCell and conditionNameCell could be made smarter to respect a min-width style (figure 3). An alternative is maybe exposing a css variable defining the min-width.

**Comment by Animmaniac** [comment]
![events-actionnameminwidth](https://user-images.githubusercontent.com/29487913/27237891-b0fa2e50-52c2-11e7-8a38-4dcabe8136c5.png)

This happens because I added a bit more padding to the left of conditions/actions names to separate them better (see figure 1). It seems the minimum size for actionNameCell or conditionNameCell is hard-coded somewhere.  If I set a min-width for actionNameCell it doesn't seem to be respected (the dividing border also disappears when collapsed).

To solve it I can reset the padding to the original value (figure 2) at the expense of a more balanced layout (although I think someone else will run into the same problem when customizing themes), or the resizing for actionNameCell and conditionNameCell could be made smarter to respect a min-width style (figure 3). An alternative is maybe exposing a css variable defining the min-width.
